universal-application-tool-0.0.1/app/services/applicant/ApplicantService.java

```diff
@@ -88,4 +88,6 @@
 
   /** Return the email of the given applicant id if they have one. */
   CompletionStage<Optional<String>> getEmail(long applicantId);
+
+  ImmutableList<Application> getAllApplications();
```

Comment:
suggestion (blocking): please document this public method

Reply:
Done.